content/installation/migrating-manager.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: bt_wiki
+title: Reinstalling and Migrating Cloudify Manager
+category: Installation
+draft: true
+weight: 400
+
+---
+
+To move your Cloudify Manager to another host, you must use same process as a reinstall. One reason to migrate is to move your Cloudify Manager to a host with more resources. You can do this in one of these ways:
+
+* **In-place reinstall** - Replace Cloudify Manager with the same version on the same host
+* **Migration reinstall** - Replace Cloudify Manager with the same version on another host
+
+The reinstall process includes:
+
+* In-place reinstall -
+
+  1. Backup a snapshot of the Cloudify Manager.
+  1. Backup the agents certificates from the Cloudify Manager.
+  1. Uninstall the Cloudify Manager from the host
+  1. Install the Cloudify Manager on the host.
+  1. Restore the snapshot of the Cloudify Manager to the host.
+
+* Migration reinstall -
+
+  1. Backup a snapshot of the Cloudify Manager.
+  1. Backup the agents certificates from the Cloudify Manager.
+  1. Install the Cloudify Manager on the host.
+  1. Restore the snapshot of the Cloudify Manager to the host.
```

Comment:
On a 4.2 snapshot, you have to wait after the snapshot restore is complete for the rest service to restart. Recommended is at least 10 seconds.